File: src/main/java/com/moxie/taosha/code_generator/dto/bean/Attribute.java
package com.moxie.taosha.code_generator.dto.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @ClassName Attribute
 * @Description Bean类型
 * @Author 淘沙
 * @Date 2018/8/23 19:35
 * @Version 1.0
 **/
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Attribute {
    /**
     * 是否不为空
     */
    String isNotNull;
    /**
     * bean 注释
     */
    String notes;
    /**
     * 字段类型
     */
    String type;
    /**
     * 字段名称
     */
    String name;
}

File: src/main/java/com/moxie/taosha/code_generator_kotlin/bean/Attribute.kt
package com.moxie.taosha.code_generator_kotlin.bean

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor

@Data
class Attribute{

        constructor()

        constructor(isNotNull: String?, notes: String?, type: String?, name: String?) {
                this.isNotNull = isNotNull
                this.notes = notes
                this.type = type
                this.name = name
        }

        /**
        * 是否不为空
        */
        internal var isNotNull: String? = null
        /**
        * bean 注释
        */
        internal var notes: String? = null
        /**
        * 字段类型
        */
        internal var type: String? = null
        /**
        * 字段名称
        */
        internal var name: String? = null
}